exotica/src/Problems/TimeIndexedSamplingProblem.cpp

```diff
@@ -102,13 +105,29 @@ void TimeIndexedSamplingProblem::Instantiate(TimeIndexedSamplingProblemInitializ
     }
 }
 
+Eigen::VectorXd TimeIndexedSamplingProblem::applyStartState()
+{
+    scene_->setModelState(startState, T_start_);
+    return scene_->getControlledState();
+}
+
```

Comment:
Move T_start_ into the `PlanningProblem` and use the implementation above for `PlanningProblem::applyStartState()`